<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WIfaceCfgBase</class>
 <widget class="QWidget" name="WIfaceCfgBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="TextLabel2">
       <property name="text">
        <string>Button text:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="edtText" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="TextLabel3">
     <property name="text">
      <string>Button tip:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MultiLineEdit" name="edtTip" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="TextLabel3_2">
     <property name="text">
      <string>Forecast tip:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MultiLineEdit" name="edtForecastTip" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QWidget</extends>
   <header>simgui/editfile.h</header>
  </customwidget>
  <customwidget>
   <class>MultiLineEdit</class>
   <extends>QWidget</extends>
   <header>simgui/editfile.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
